core-common/src/main/java/org/mapstruct/Mapper.java

```diff
@@ -160,6 +160,16 @@
      */
     NullValueCheckStrategy nullValueCheckStrategy() default ON_IMPLICIT_CONVERSION;
 
+    /**
+     * Determines whether to use field or constructor injection. This is only used on annotated based component models
+     * such as CDI, Spring and JSR330.
+     *
+     * Can be overridden by the one on {@link MapperConfig}.
```

Comment:
Fxd.

Would be great if you can help a bit with the test.

Sure, so this PR will be on hold for a while, right?